.lintstagedrc

```diff
@@ -1,5 +1,5 @@
 {
   "*.scss": ["stylelint --fix --color --config .stylelintrc.json"],
-  "src/*.{js,jsx,ts,tsx}": ["eslint --fix"],
-  "src/**/*.{js,jsx,ts,tsx}": ["eslint --fix"]
+  "src/*.{js,jsx,ts,tsx}": ["eslint --fix", "rollup --config rollup.config.js --quiet"],
+  "src/**/*.{js,jsx,ts,tsx}": ["eslint --fix", "rollup --config rollup.config.js --quiet"]
```

Comment:
My one concern here is that you are able to commit the staged changes successfully. Everything seems to happen without issue (lint is all checkmarks), but more action is actually required from the developer. For users with some sort of terminal setup that allows them to see the status of their branch, this might be ok as they'll notice the uncommitted changes. However, it's definitely not the cleanest implementation.

Did you consider always running the compilation step as a part of `pre-commit`? Since it's just a bash script, we can create some more custom logic without being constrained to a `yarn` script. For example, we could leave `.lintstagedrc` as is but change `pre-commit`:

```
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npx lint-staged
cd src
if [[ `git status . --porcelain` ]]; then
  echo "Detected changes in JS src files. Running JS compilation"
  yarn rollup --config rollup.config.js
fi
cd ../dist
if [[ `git status . | grep "not staged"` ]]; then
  echo "Detected unstaged changes in JS dist files. Please add changed files"
  exit 1
fi
```

Here's a loom covering the new behavior:
https://www.loom.com/share/a7b9deb0d3524c70a18aff118350f246

Reply:
@drew-beckmen  😍 Love it, _much_ better solution, thank you!! Do you want to take over this branch and push those changes up to this PR?

My only additional suggestion would be looking at making the warning text red (and maybe some blank lines around it?) to make sure devs notice it... inlining ANSI color codes feels super icky, but I think will get the job done -- [ex 1](https://codeburst.io/colorize-your-shell-commands-without-any-libraries-or-plugins-2cd4a264a17e), [ex 2](https://www.codegrepper.com/code-examples/shell/bash+echo+color).

Reply:
sure! happy to push up the changes

Reply:
updated script output:
![Screen Shot 2022-05-25 at 2 12 33 PM](https://user-images.githubusercontent.com/17502635/170336951-d4d1725d-315c-4175-8590-243fd1644fce.png)